manifests/ntp.pp

```diff
@@ -7,64 +7,71 @@
 
   # Get facts for distro/release
   $distro = $facts['os']['distro']['id']
-  $release = $facts['os']['distro']['release']['full']
+  $release = $facts['os']['distro']['release']['major']
 
-  # Help Puppet understand to use systemd for Ubuntu 16.04 hosts
-  if $distro == 'Ubuntu' and versioncmp($release, '15.04') >= 0 {
-    Service {
-      provider => 'systemd',
+  # Only do anything on older Ubuntu and Debian. For later versions
+  # sunet::server will use sunet::chrony instead. This if-statement is needed
+  # because not every ops-repo uses sunet::server to select what ntp class to
+  # use. If changing this if-statement you might need to update it in
+  # sunet::server as well.
+  if ($distro == 'Ubuntu' and versioncmp($release, '22.04') <= 0) or ($distro == 'Debian' and versioncmp($release, '11') <= 0) {
```

Comment:
Can we make a fact out of this, in sunet::ntp and use that in sunet::server, so if this fact is set, we do one thing, and another if the fact is unset? Meaning we can detect if the sunet::ntp package has been pulled in from the side?

Reply:
I have been discussing this with patlu and I think maybe syncing the if clause is a better way to go, facts can be finicky to get right too.

Reply:
I guess we could, i agree keeping things in sync across two files by hand is not a great approach, however i don't feel it is necessarily worth the work of creating a custom fact script at this point as I dont think we will go back to ntpd and sunet::ntp will fade into the background the more we use never machines.

Reply:
... on the other hand, "lets do it right", now fact has been commited. Thanks for the pointer!